dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/MaintenanceController.java

```diff
@@ -246,57 +231,45 @@ public void clearCache()
     @RequestMapping( value = "/dataPruning/organisationUnits/{uid}", method = { RequestMethod.PUT,
         RequestMethod.POST } )
     @PreAuthorize( "hasRole('ALL')" )
-    @ResponseStatus( HttpStatus.NO_CONTENT )
```

Comment:
Note that `@ResponseStatus( HttpStatus.NO_CONTENT )` were `WebMessageUtils.ok` are inconsistent. 
This is the same for a number of methods in this controller which now all return `OK` since they do have a response body (the `WebMessage`).